Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+importlib_metadata = "*"
+rich = "*"
+requests = "*"
+pandas = "*"
+ipython = "*"
+
+[dev-packages]
+mypy = "*"
+pytest = "*"
+flake8 = "*"
+pre-commit = "*"
+pytest-cov = "*"
+twine = "*"
+setuptools-scm = "*"
+flake8-bugbear = "*"
+flake8-docstrings = "*"
+pytest-env = "*"
+types-requests = "*"
+
+[requires]
+python_version = "3.10"
```

Comment:
I'm not super familiar with `pipenv` would this have a similar effect as `python_requires=="3.10"` in `setup.py`? Currently, we support 3.8 - 3.10. Just want to make sure we're not dropping 3.8 and 3.9 for the moment.

Reply:
The version specified there is the one that the developer of the application uses when making contributions. The testing regime intends to ensure that the tests pass and everything works in other versions.

Reply:
Perfect!